docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
     ports:
       - "9983:8983"
     volumes:
-      - data:/opt/solr/server/solr/mycores
```

Comment:
TBH, I have no idea why this fixed it, but I was able to start solr, create a config, stop solr, start it, then search for the config. I thought the issue was with the entry point originally.

Reply:
I didn't have much luck finding out what happens differently with these 2 paths either when observing the service's logs.

Reply:
I have seen strange behavior (permission issues, etc.) if one attempts to mount a volume in a directory that already exists?  Is that the case here?

Reply:
Should we be adding that deployment constraint that Chris mentioned to ensure we're accessing the same volume on the same host every time?

Reply:
Yeah I believe that `/var/solr` is the default directory in the solr image.